doc/src/sphinx/SBTPlugin.rst

```diff
@@ -5,6 +5,8 @@ Add a line like this to your `project/plugins.sbt` file:
 
 ::
 
+    resolvers += "Twitter's Repository" at "http://maven.twttr.com/"
```

Comment:
I'm sure there are various instances of this around, but it should move to https now.